integration/setup/setup.go

```diff
@@ -0,0 +1,89 @@
+// +build k8srequired
+
+package setup
+
+import (
+	"log"
+	"os"
+	"testing"
+
+	"github.com/giantswarm/e2e-harness/pkg/framework"
+	"github.com/giantswarm/e2etemplates/pkg/e2etemplates"
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/kvm-operator/integration/env"
+	"github.com/giantswarm/kvm-operator/integration/teardown"
+	"github.com/giantswarm/kvm-operator/integration/template"
+)
+
+const (
+	kvmResourceValuesFile = "/tmp/kvm-operator-values.yaml"
+)
+
+// WrapTestMain setup and teardown e2e testing environment.
+func WrapTestMain(g *framework.Guest, h *framework.Host, m *testing.M) {
+	var r int
+
+	err := Setup(g, h)
+	if err != nil {
+		log.Printf("%#v\n", err)
+		r = 1
+	} else {
+		r = m.Run()
+	}
+
+	if env.KeepResources() != "true" {
+		teardown.Teardown(g, h)
+	}
+
+	os.Exit(r)
+}
+
+// Setup e2e testing environment.
+func Setup(g *framework.Guest, h *framework.Host) error {
+	var err error
+
+	err = h.Setup()
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	err = Resources(g, h)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	// TODO(r7vme): Enable, when real kvm host cluster will be used.
```

Comment:
What about this?

Reply:
we don't setup any guest clusters at the moment. So to prevent failure below i've commented out setup of guest client.

```
{"caller":"github.com/giantswarm/kvm-operator/vendor/github.com/giantswarm/e2e-harness/pkg/framework/guest.go:202","level":"debug","message":"k8s API is still down: wait","time":"2018-07-24T14:55:39.852588+00:00"}
```